src/index.js

```diff
@@ -53,10 +55,51 @@ const updateTask = () => {
     list[i].addEventListener('change', () => {
       storage.array[i].description = list[i].value;
       const stringData = JSON.stringify(storage.array);
-      localStorage.setItem('tasks', stringData);
+      localStorage.setItem('Tasks', stringData);
       window.location.reload();
     });
   }
 };
 updateTask();
-export default storage;
\ No newline at end of file
+for (let i = 0; i < storage.array.length; i += 1) {
+  const checkbox = document.getElementById(`box${i}`);
+  checkbox.addEventListener('change', () => {
+    if (storage.array[i].complete === false) {
+      storage.array[i].complete = true;
+      const stringData = JSON.stringify(storage.array);
+      localStorage.setItem('Tasks', stringData);
+      const list = document.getElementById(`list${i}`);
+      list.style.textDecoration = 'line-through';
+    } else if (storage.array[i].complete === true) {
+      storage.array[i].completed = false;
+      const stringData = JSON.stringify(storage.array);
+      localStorage.setItem('Tasks', stringData);
+      const lists = document.getElementById(`list${i}`);
+      lists.style.textDecoration = 'none';
+    }
+  });
+}
+const clearm = document.getElementById('clear');
+clearm.addEventListener('click', () => {
+  const filtered = storage.array.filter((items) => items.completed === false);
+  const stringData = JSON.stringify(filtered);
+  for (let i = 0; i < storage.array.length; i += 1) {
+    const listedItem = document.getElementById(`item${i}`);
+    listedItem.remove();
+  }
+  localStorage.setItem('Tasks', stringData);
+  const rmv = document.querySelector('.trashIcon');
+  rmv.style.display = 'none';
+  updateTask();
+  updateIndex();
+});
```

Comment:
## Optional changes ♻️ 
The previous reviewer did not spot that but I highly recommend you fix it 👍. 

- [ ] Currently, when I mark todos as completed and click on the clear all completed button, it removes the todos from the screen but when I refresh the browser, the todos get loaded again. This means you are just removing them from the screen but not on localStorage. It would be nice if you remove the completed todos from the screen and from localStorage. Kindly work on that.

Reply:
Thank you so much @AmaduKamara, I will make sure it  fixed